src/components/Layout/styles.js

```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
@sbussard do you know if we're going to run into another async issue like last time LOL

Reply:
I have no idea LOL